js/bit2c.js

```diff
@@ -747,14 +752,24 @@ module.exports = class bit2c extends Exchange {
             'order': orderId,
             'type': undefined,
             'side': side,
-            'takerOrMaker': undefined,
+            'takerOrMaker': makerOrTaker,
             'price': price,
             'amount': amount,
             'cost': undefined,
             'fee': fee,
         }, market);
     }
 
+    removeCommaFromValue (str)  {
```

Comment:
@JaCoderX two things, first the formatting is wrong, do you have eslint properly configured?

Second, you version is not transpilable to PHP, I think you need to do something like this:

```Javascript
    removeCommaFromValue (str) {
        let newString = '';
        const strParts = str.split (',');
        for (let i = 0; i < strParts.length; i++) {
            newString += strParts[i];
        }
        return newString;
    }
``

Reply:
I use mostly python and very little with js.
made a basic setup just for the PR

but will look into eslint